### DISCUSSION TITLE
Como começar a estudar programação?

### DISCUSSION THREAD

**Discussion by KennedyAbrante** [frontendbr/forum#1533] [Discussão]
Boa Noite Galera me chamo Kennedy descobri faz pouco tempo esse universo de programação e estou querendo muito aprender sobre, para criar softwares e trabalhar na área 
tenho muitas duvidas e estou aprendendo sobre o assunto por videos . mas gostaria de colocar a mão na massa e ja começar . Por onde começar a Estudar ?

**Comment by yuritoledo** [comment]
@KennedyAbrante Dá uma olhada nos cursos gratuitos da rocketseat: https://rocketseat.com.br/starter

é um bom lugar pra começar.

**Comment by theandersonn** [comment]
E aí @KennedyAbrante, blz?!
Dê uma olhada nesses tópicos https://github.com/frontendbr/forum/issues/1518, https://github.com/frontendbr/forum/issues/1515, https://github.com/frontendbr/forum/issues/1421

Vai postando suas dúvidas!

**Comment by HmmerHead** [comment]
Seria interessante buscar informações mais gerais sobre a área. Já deram alguns bons link's.
Gosto, de vez enquanto, olhar essa rota ([Developer Roadmap](https://github.com/kamranahmedse/developer-roadmap)) e ver como estão as coisas. 
Após essa busca, saiba qual é o seu caminho e começe a se aprofundar

Tem esse Canal que acho bacana também - [Código Fonte TV](https://www.youtube.com/user/codigofontetv)

**Comment by victorpothin** [comment]
Uma coisa muito legal é praticar, começar com coisas simples para que a complexidade não te engula. O pessoal já passou ótimos links e um roadmap.
Outra pratica legal seria você tentar em sua linguagem favorita ou área em desenvolvimento(vide roadmap) resolver algum pequeno e simples problema que te inflige no dia a dia. Além de você conhecer muito sobre o problema poderá se apaixonar por ele e resolve-lo.

**Comment by Guilherme-Artigas** [comment]
Ola glra, boa tarde, obrigado pelas resposta acima, desculpe se isso ficar extenso!

estou iniciando nesse universo, tenho todas as duvidas que possam existir sobre o assunto, menos do que quando comecei a estudar kkk... mais ainda são muitas, eu tenho como objetivo aprender a programar para ter uma profissão e mudar a minha vida, isso pra mim é tão grande que chega a ser um sonho, trabalhar como desenvolvedor, seja autônomo ou em alguma empresa...

Vejo muitas tecnologias no mercado, olhar pra tudo isso quando se esta começando é aterrorizante, por no seu dia-a-dia vc acaba fazendo aquela pesquisa rápida, sobre como anda as vagas do meu curso, e acabamos nos deparando com vagas extremamente exigentes, enfim... isso não é o suficiente pra me fazer desistir, mais me deixa um pouco preocupado! 

Minhas Preocupações...
Eu sei bem pouco do inglês, algumas palavras somente, ate que ponto isso dificulta minha vida, ou não tem interferência alguma? (isso só facilita a apredinzagem?!) (Ja ouvi falar que vai pegando com o tempo quando se trabalha com programação, é verdade?)
Uma grande dificuldade é por em pratica aquele conteúdo que assisto, alguém poderia indicar algum lugar que tenha projetos para Iniciantes, coisas bem simples, para praticas das Tags, Seletores e Propriedades iniciais! uma comunidade que tenha gente perguntando sobre HTML5 CSS3 a todo momento

**Comment by HmmerHead** [comment]
Inglês é importante. Ajuda bastante a desenvolver, porém é possível aprender com o que temos em nossa língua. Determinados assuntos são bem abordados por brasileiros, mas podem existir casos  que indivíduos que falam outro idioma abordarão mais.

Por isso, inicialmente, enquanto você estiver começando em um conteúdo novo, fique tranquilo, mas tenha em mente que poderá ser necessário de acordo com as sua dúvidas e necessidades.
 
No meu caso, achei bastante material em Português, como cursos online e artigos, porém consegui um melhor entendimento quando encontrei em Inglês. Isso não é uma regra, é claro. vai depender de você, e de quem está repassando o conteúdo.

Quanto as suas dificuldades em aplicar, sugiro tentar [isso](https://github.com/felipefialho/frontend-challenges) aqui ou buscar por mais pelo Git

**Comment by TL-Developer** [comment]
Da uma procurada la no youtube, onde tudo começou fazer sentido pra mim:
Universidade XTI
Me ajudou bastante
